basic/okta_sdk/okta_sdk.tf

```
@@ -12,13 +12,13 @@ resource "sym_integration" "okta" {
 }
 
 # An AWS Secrets Manager Secret to hold your Okta API Key. Set the value with:
-# aws secretsmanager put-secret-value --secret-id "main/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
+# aws secretsmanager put-secret-value --secret-id "${local.environment_name}/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
 resource "aws_secretsmanager_secret" "okta_api_key" {
-  name        = "main/okta-api-key"
+  name        = "${local.environment_name}/okta-api-key"
   description = "API Key for Sym to call Okta APIs"
 
   tags = {
-    # This SymEnv tag is required and MUST match the SymEnv tag in the 
+    # This SymEnv tag is required and MUST match the SymEnv tag in the
     # aws_iam_policy.secrets_manager_access in your `secrets.tf` file
```

```suggestion
    # This SymEnv tag is required and MUST match the `environment` variable
    # passed into the `secrets_manager_access` module in your `secrets.tf` file
```

After updating secrets.tf, we should also update this comment